Classes/common/Encryption/Keychain/CDTEncryptionKeychainManager.h

```diff
@@ -0,0 +1,76 @@
+//
+//  CDTEncryptionKeychainManager.h
+//  CloudantSync
+//
+//  Created by Enrique de la Torre Fernandez on 09/04/2015.
+//  Copyright (c) 2015 IBM Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "CDTEncryptionKeychainStorage.h"
+
+/**
+ Use this class to generate a Data Protection Key (DPK), i.e. a strong password that can be used
+ later on for other purposes like ciphering a database.
```

Comment:
I'm not sure "ciphered" is in common use in this context, though is probably technically accurate. I'd use "encrypted", "encrypting" etc.